intro-HPC/ch_account.tex

```diff
@@ -82,28 +82,29 @@ \subsection{How do SSH keys work?}
   need to be installed on your Windows machine first, before we can start the
   actual work.
 
-  \subsection{Install PuTTY: A free telnet/SSH client}
+  \subsection{Get PuTTY: A free telnet/SSH client}
   \label{sec:install-putty}
 
-  We recommend to use the PuTTY tools package, which is freely available. You
-  do not need to install PuTTY, you can download the PuTTY and PuTTYgen executable
+  We recommend to use the PuTTY tools package, which is freely available.
+
+  You do not need to install PuTTY, you can download the PuTTY and PuTTYgen executable
   and run it. This can be useful in situations where you do not have the
   required permissions to install software on the computer you are using.
   Alternatively, an installation package is also available.
 
   You can download PuTTY from the official address:
-  \url{http://www.chiark.greenend.org.uk/\~sgtatham/putty/}
+  \url{https://www.chiark.greenend.org.uk/~sgtatham/putty/latest.html}. You
+  probably want the 64-bits version. If you can install software on your computer,
+  you can use the ``Package files'', if not, you can download and use \verb|putty.exe|
+  and \verb|puttygen.exe| in the ``Alternative binary files'' section.
```

Comment:
Does this section have a dedicated URL? If so, include it.

Reply:
It doesn't: it's one page. Users can even see it without scrolling down (https://www.chiark.greenend.org.uk/~sgtatham/putty/latest.html)